cla-backend-go/v2/company/service.go

```diff
@@ -737,3 +785,21 @@ func (s *service) getCompanyAndProjects(companySFID, projectSFID string) (*v1Mod
 	}
 	return comp, projects, nil
 }
+
+func sendEmailToUserCompanyProfile(orgName string, userEmail string, username string, LFXPortalURL string) {
+	subject := "EasyCLA: EasyCLA: Invitation to create LFID and complete process of becoming CLA Manager"
+	recipients := []string{userEmail}
+	body := fmt.Sprintf(`
+<p>Hello %s,</p>
+<p>This is a notification email from EasyCLA regarding the newly created Salesforce Organization %s .</p>
+<p>The organization profile can be completed via <a href="%s/company/manage/" target="_blank">clicking this link</a>
```

Comment:
We are missing the standard last paragraph. See: https://docs.google.com/document/d/1jFlSq1tx66LbBwGbSfJm3QS4st68GblfzBEMZRya9gk/edit

For example:
<p>If you need help or have questions about EasyCLA, you can
<a href="https://docs.linuxfoundation.org/docs/communitybridge/communitybridge-easycla" target="_blank">read the documentation</a> or 
<a href="https://jira.linuxfoundation.org/servicedesk/customer/portal/4/create/143" target="_blank">reach out to us for
support</a>.</p>
<p>Thanks,</p>
<p>EasyCLA support team</p>`,